### PR TITLE
[PWX-33954] Added test for PoolExpandResizeClusterNoQuorum

### PR DESCRIPTION
**Pull request by rkinhalkar-px**
Added test for PoolExpandResizeClusterNoQuorum

PWX-33954

Testing done :- 
https://jenkins.pwx.dev.purestorage.com/job/Users/job/Rushikesh/job/rushi_pxdv-7-dev/103/

### CODE REVIEW

File: tests/basic/pool_function_test.go [hunk 1]
```diff
@@ -504,3 +505,112 @@ var _ = Describe("{PoolExpandWhileResizeDiskInProgress}", func() {
...
 
 })
+
+var _ = Describe("{PoolExpandResizeClusterNoQuorum}", func() {
+	//1) Deploy px with cloud drive.
+	//2) Make Cluster out of quorum
+	//3) Expand a healthy pools by resize-disk
+
+	var testrailID = 34542845
+	// testrailID corresponds to: https://portworx.testrail.net/index.php?/cases/view/34542845
```
Comment (vinayakshnd) [inline tests/basic/pool_function_test.go:515]: For some reason, this testrail link is not opening actual testrail testcase

File: tests/basic/pool_function_test.go [hunk 2]
```diff
@@ -504,3 +505,112 @@ var _ = Describe("{PoolExpandWhileResizeDiskInProgress}", func() {
...
+				err := Inst().V.StopDriver([]node.Node{n}, false, nil)
+				log.FailOnError(err, "error stopping driver on node %s", n.Name)
+
+				err = Inst().V.WaitDriverDownOnNode(n)
+				log.FailOnError(err, "error while waiting for driver down on node %s", n.Name)
+				driverDownNodes = append(driverDownNodes, n)
+				i += 1
+			}
+		})
+
```
Comment (vinayakshnd) [inline tests/basic/pool_function_test.go:587]: At this stage, let test ensure that cluster has actually went out of quorum using one of volume driver API.

For this, below API can be invoked for node, which is non-kvdb.
https://github.com/portworx/torpedo/blob/master/drivers/volume/volume.go#L297

File: tests/basic/pool_function_test.go [hunk 3]
```diff
@@ -504,3 +505,112 @@ var _ = Describe("{PoolExpandWhileResizeDiskInProgress}", func() {
...
+		stepLog = fmt.Sprintf("Expanding pool on kvdb node using resize-disk")
+		Step(stepLog, func() {
+
+			log.FailOnError(err, fmt.Sprintf("Failed to get pool using UUID %s", selPool.Uuid))
+			expectedSize := poolToBeResized.TotalSize * 2 / units.GiB
+
+			log.InfoD("Current Size of the pool %s is %d", selPool.Uuid, poolToBeResized.TotalSize/units.GiB)
+			err = Inst().V.ExpandPool(selPool.Uuid, api.SdkStoragePool_RESIZE_TYPE_RESIZE_DISK, expectedSize, true)
+			dash.VerifyFatal(err, nil, "Pool expansion init successful?")
+			Step("set cluster to running", func() {
```
Comment (vinayakshnd) [inline tests/basic/pool_function_test.go:597]: Lets start Portworx on nodes after actually Pool is completely resized. i.e. after `waitForPoolToBeResized` call is successfull.

File: tests/basic/pool_function_test.go (hunk 1 above)
Comment (rkinhalkar-px) [reply tests/basic/pool_function_test.go:515]: done

File: tests/basic/pool_function_test.go (hunk 2 above)
Comment (rkinhalkar-px) [reply tests/basic/pool_function_test.go:587]: done

File: tests/basic/pool_function_test.go (hunk 3 above)
Comment (rkinhalkar-px) [reply tests/basic/pool_function_test.go:603]: done

**Pull request closed without merging**